src/dotnet/Chat.Service/ChatsMigrationBackend.cs

```diff
@@ -0,0 +1,449 @@
+                                .SetProperty(c => c.DbAuthorId, c => migratedAuthor.NewId),
+                            cancellationToken)
+                        .ConfigureAwait(false);
+                }
+
+                await dbContext.SaveChangesAsync(cancellationToken).ConfigureAwait(false);
+            }
+        }
+
+        if (dbAuthors.Count > 0) {
+            updateCount = await dbContext.Authors
+                .Where(c => c.ChatId == chatSid)
+                .ExecuteDeleteAsync(cancellationToken)
+                .ConfigureAwait(false);
+            AssertXUpdated(updateCount, dbAuthors.Count);
+
+            Log.LogInformation("Updated {Count} author records", dbAuthors.Count);
+        }
+    }
+
+    private async Task UpdateChatEntries(
+        ChatDbContext dbContext,
+        string chatSid,
+        ChatId newChatId,
+        IReadOnlyCollection<MigratedAuthor> migratedAuthors,
+        CancellationToken cancellationToken)
+    {
+        int updateCount;
+        var chatEntryAuthorSids = await dbContext.ChatEntries
+            .Where(c => c.ChatId == chatSid)
+            .Select(c => c.AuthorId)
+            .Distinct()
+            .ToListAsync(cancellationToken)
+            .ConfigureAwait(false);
+
+        foreach (var authorSid in chatEntryAuthorSids) {
+            var authorId = new AuthorId(authorSid);
+            AuthorId newAuthorId;
+            if (authorId.LocalId > 0)
+                newAuthorId = migratedAuthors.First(c => OrdinalEquals(c.OriginalAuthor.Id.Value, authorSid)).NewId;
+            else if (authorId.LocalId == Constants.User.Walle.AuthorLocalId)
+                newAuthorId = new AuthorId(newChatId, authorId.LocalId, AssumeValid.Option);
+            else
+                throw StandardError.Constraint($"Unexpected author local id. Local id is '{authorId.LocalId}'.");
+
+            var newAuthorSid = newAuthorId.Value;
+            updateCount = await dbContext.ChatEntries
+                .Where(c => c.ChatId == chatSid)
+                .Where(c => c.AuthorId == authorId)
+                .ExecuteUpdateAsync(s => s.SetProperty(c => c.AuthorId, newAuthorSid), cancellationToken)
+                .ConfigureAwait(false);
+            AssertAtLeastOneUpdated(updateCount);
+
+            Log.LogInformation("Updated AuthorId for {Count} chat entry records with author id '{AuthorId}'." +
+                " New author id is '{NewAuthorId}'", updateCount, authorSid, newAuthorSid);
+        }
+
+        for (int i = 0; i < 2; i++) {
+            var entryKind = (ChatEntryKind)i;
+            var prefix = newChatId + ":" + i + ":";
+            updateCount = await dbContext.ChatEntries
+                .Where(c => c.ChatId == chatSid)
+                .Where(c => c.Kind == entryKind)
+                .ExecuteUpdateAsync(setters => setters
+                        .SetProperty(c => c.Id, c => prefix + c.LocalId)
```

Comment:
Wrong double indent.

Reply:
I think it's ok. Look to the entire statement. Also it's how rider reformat it's automatically.